wishlist.go

```diff
@@ -107,7 +123,7 @@ func (m *ListModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			if selectedItem == nil {
 				return m, nil
 			}
-			cmd := m.client.For(selectedItem.(*Endpoint))
+			cmd := m.client.For(selectedItem.(*ItemWrapper).endpoint)
```

Comment:
The interface conversion here causes a runtime panic. `selectedItem` is `wishlist.ItemWrapper`, not `*wishlist.ItemWrapper`